### ISSUE TITLE
Disable cookies globally and set defaults for all of a request’s fields

### ISSUE THREAD

**Issue by ocroz** [a85/POSTMan-Chrome-Extension#11008]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Having to individually disable the cookie jar for every single request I create is tiresome.
See https://community.postman.com/t/feature-request-allow-disabling-redirects-and-cookies-globally/28296.

### Describe the solution you'd like

Please consider allowing users to set defaults for all of a request’s fields, those that are visible from the Settings tab.

### Describe alternatives you've considered

The workaround I am using at the moment is duplicating a “template” request that already has the settings I need set, but I started doing this after building up a collection of many requests. So I still need to go through each request one by one and change their individual settings.

### Additional context

![444b52111df2eef6f52a1a6efaacd0ffcb81e938](https://user-images.githubusercontent.com/29784345/173868041-e3c1e256-79cb-49a1-91d7-1031b9f9cd37.gif)

**Comment by Sesna** [comment]
yeah, I want this feature too, I don't want the cookie to be set by default, I use it for testing and need to ensure each request sends the same data, if I need to attach the cookie, I'll add it manually.

**Comment by holdenrehg** [comment]
+1

**Comment by mattsmac** [comment]
+1 Agreed.  I need to disable the cookie jar and the follow redirects flag for every request.  It would be a huge time-saver to have these options at the collection level.

**Comment by homestar9** [comment]
+1

**Comment by tijsverkoyen** [comment]
+1

**Comment by markslWix** [comment]
+1

**Comment by alobaili** [comment]
Preferably there should be a global setting in the Postman Settings screen to completely disable the cookie jar.
Another option is to develop a Settings tab in the root of the collection along side the collection Variables and Runs tabs, that includes an option to disable the cookie jar for all requests inside the collection, and use the existing per-request option to override the option on the collection level. Similar to how Authorization tab is currently behaving (requests can override the Authorization inherited from the parent collection).

**Comment by franzvezuli** [comment]
+1. Need the ability to disable this globally. For `Authorization` header testing, the cookie can sometimes take precedence. Turning them off per request is a huge time waster.

This makes Postman effectively useless for non-cookie based authentication API testing.

**Comment by ibraheemalayan** [comment]
+1

**Comment by stevesutton-hds** [comment]
> This makes Postman effectively useless for non-cookie based authentication API testing.

  You're not kidding!  This fsck'ed me up for hours trying to figure out why my server wasn't changing the session ID.  Turns out that Postman was re-submitting the session Id  - ***WITHOUT MY CONSENT*** - in every request!

**Comment by ErcinDedeoglu** [comment]
Postman is dead.

**Comment by Webappmaster-Eugene** [comment]
I join the problem! Please enable/disable this functionality at the Postman collection level.

**Comment by ErcinDedeoglu** [comment]
They're deleting my review/comment instead of offering a solution. If it's a paid feature, let us know. They're ignoring us, and I don't understand how this tool became popular without this option.

**Comment by ErcinDedeoglu** [comment]
By the way, there are other alternatives that provide this option: Insomnia and HTTPie. The only way to put pressure on such tools is by switching to another that actually works. This was my comment, which was deleted. I'm going to take a screenshot of this one also; if this gets deleted, then I'll publish a Medium article and see how they delete it again.

**Comment by gabmule** [comment]
I see this problem alive since 2018 so it is officially ignored, and it is 100% dumb and infuriating. this is a REST client, the default should be NO COOKIES, the S in REST is for STATELESS.
I set up a script for a collection to populate automatically the bearer token for all following requests, but it doesn't work because the login service returns error as the cookie is expired (it is not supposed to be there in the first place), and there is no way to even remove it, let alone tell postman to please stop doing it. goodbye forever postman, you were useful up until corporate greed changed you. fuck you

**Comment by mikatuo** [comment]
Wow this is such a basic and needed option for a tool that was designed for API testing. On top of that even when deleting a cookie it does not always resets it. I was fighting the cookie header for 10 minutes and could not clear it consistently enough to share steps with a team member. It is buggy. Hard to believe this feature request has been sitting for years…

**Comment by mandras73** [comment]
+1
I'm using pre-script to add cookies with pm.request.headers
And for this, I'll need to turn off all requests' cookie jars one-by-one... why is this not a collection level option?

**Comment by liranzairi** [comment]
Joining the request. Why is it being ignored?

**Comment by ayushkr12** [comment]
+1

**Comment by skylarshaffer** [comment]
+1

**Comment by scottkingdon** [comment]
+1 

This is a huge annoyance.

**Comment by caio-pizzol** [comment]
+1

**Comment by Baldovie** [comment]
+1.
This is now 3 years old and consistent nudges to request the function.
And a few posts deleted by the repo owners is suspicious but also reflects that this thread has been acknowledged and just ignored.

**Comment by timothy-whitehouse** [comment]
We had a collection that did not have every "Disable cookie jar" set.  It caused many 401 Unauthorized errors when some changes were made to our back end code.  We did not realize what was happening.  After many hours of research, I found that these failing tests all had "Disable cookie jar" set to disabled under the Settings.  So, I had to spend about 2 hours going through all of the hundreds of individual tests, clicking on the test, then on the Settings, then the check mark for "Disable cookie jar", over and over and over, to finally turn on the setting for "Disable cookie jar" for all of the hundreds of tests we have.  This is a painstaking task.  Please prioritize this.  You can see now that 42 other people want this as well.  I am sure there are many more than this who also want it, but just did not find this thread yet.  

In fact, you could take it a step further later and allow more control over all of the Settings to be able to apply them to every individual test.  I am not asking for this out of the gate.  But, anything else that would apply under the Settings tab would fall into this category of being a pain staking operation to perform hundreds of times in a row.   Such other items like http redirect, follow direct, SSL, etc. could all be in this similar boat.

For now, I would be happy with just the disable cookie jar being global in some way.

Side note: programmatically disabling the cookie jar in code, globally, at the high level folders, or even the top folder for the entire collection, does not seem to work either.  I believe each individual test overrides that.  This is another reason to make this global.

**Comment by jonathanhaviv** [comment]
Hey all, thanks for the continued follow-up over the years. We're going to look at adding support for disabling cookies locally. Will let you know once we have more updates!

**Comment by jorgetov** [comment]
Please make it happen!

**Comment by omasseau** [comment]
+1. Having to disable cookies jar on each indivual request when you have hundred of them is just horrible user experience.

**Comment by jonathanhaviv** [comment]
Hey all thanks for following up, we should have an update in the next week or two. Sorry about any delays!

**Comment by binoy-d-shah** [comment]
Please please make it happen. Any update?

**Comment by jonathanhaviv** [comment]
Yes we do! Sorry for any delays but excited to announce that in version 11.70.6 we have added this setting globally! You can find it with other request settings by clicking the gear icon in the top right! I'm going to close this issue for now, but let us know if you run into anymore issues.

<img width="533" height="96" alt="Image" src="https://github.com/user-attachments/assets/19c99a0f-bb8d-48db-b0ff-914b9a8c0e10" />
